deepmd/utils/tabulate.py

```diff
@@ -82,20 +82,31 @@ def __init__(self,
         self.sub_graph, self.sub_graph_def = self._load_sub_graph()
         self.sub_sess = tf.Session(graph = self.sub_graph)
 
-        try:
-            self.sel_a = self.graph.get_operation_by_name('ProdEnvMatA').get_attr('sel_a')
-            self.prod_env_mat_op = self.graph.get_operation_by_name ('ProdEnvMatA')
-        except Exception:
-            self.sel_a = self.graph.get_operation_by_name('DescrptSeA').get_attr('sel_a')
-            self.prod_env_mat_op = self.graph.get_operation_by_name ('DescrptSeA')
+        if isinstance(self.descrpt, deepmd.descriptor.DescrptSeR):
+            try:
+                self.sel_a = self.graph.get_operation_by_name('ProdEnvMatR').get_attr('sel')
+                self.prod_env_mat_op = self.graph.get_operation_by_name ('ProdEnvMatR')
+            except Exception:
```

Comment:
Use `KeyError` instead of `Exception`. https://www.tensorflow.org/api_docs/python/tf/Graph#get_operation_by_name gives what `get_operation_by_name` raises.

Reply:
OK~